example/src/main.rs

```diff
@@ -0,0 +1,26 @@
+#[diplomat::bridge]
+mod ffi {
+    use fixed_decimal::FixedDecimal;
+
+    #[repr(transparent)]
+    pub struct ICU4XFixedDecimal(pub FixedDecimal);
+
+    impl ICU4XFixedDecimal {
+        fn new(v: i32) -> Box<ICU4XFixedDecimal> {
```

Comment:
Nit: I remember in our review of this earlier, you expressed intent to move the Box into the struct

Reply:
Oh, I asked for it to be reversed: this makes reference semantics worse